packages/scss/src/components/_page-header.scss

```diff
@@ -0,0 +1,49 @@
+.pageHeader {
+	@include elevate(1);
+	padding: 2.5rem 2.5rem _theme('spacings.standard');
+	background-color: _color('white');
+
+	&.mod-withMenu {
+		padding-bottom: 0;
+	}
+
+	&.mod-withBreadcrumbs {
+		padding-top: _theme('spacings.small');
+
+		.breadcrumbs {
+			margin-bottom: 0;
+		}
+	}
+
+	.menu {
+		&::after {
+			display: none;
+		}
+	}
+}
+
+.pageHeader-content {
+	display: flex;
+	justify-content: space-between;
+	align-items: flex-start;
+	flex-wrap: wrap;
+}
+
+.pageHeader-content-title,
+.pageHeader-content-actions {
+	display: flex;
+	align-items: center;
+	flex-wrap: wrap;
+}
+
+.pageHeader-content-actions {
+	& > *:not(:last-child){
+		margin-right: _theme('spacings.small');
+	}
+}
+
+.pageHeader-description {
+	max-width: 50rem;
```

Comment:
Why 50rem ? This is a strange arbitrary value. Maybe we should link it to some kind of responsive, using % unit or make it themable ?

Reply:
this is about the recommendation on design that we must display less than 40 words i think on the same line ?